stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1309,12 +1310,19 @@ internal constructor(
         isRetrying: Boolean = false,
     ): Call<Message> {
         val relevantPlugins = plugins.filterIsInstance<SendMessageListener>().also(::logPlugins)
-        val relevantInterceptors = interceptors.filterIsInstance<SendMessageInterceptor>()
-        return CoroutineCall(scope) {
+        val sendMessageInterceptors = interceptors.filterIsInstance<SendMessageInterceptor>()
+        val prepareMessageInterceptor = interceptors.find { interceptor ->
+            interceptor is PrepareMessageInterceptor
+        } as? PrepareMessageInterceptor
+
+        val preparedMessage = getCurrentUser()?.let { user ->
```

Comment:
`getCurrentuser()` will return `null` when WS is not connected.
There is another method that obtains a user from the userCredentials (`ChatClient::getStoredUser`)
As far as we want to be able to send messages while WS is not connected, we should try to use it